packages/loader/container-definitions/src/loader.ts

```diff
@@ -185,6 +185,11 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
  * The Runtime's view of the Loader, used for loading Containers
  */
 export interface ILoader extends IFluidRouter {
+    /**
+     * If the loader can resolve external requests
+     */
+    readonly handlesExternalRequests: boolean;
```

Comment:
Not sure if this is considered "safe" post GA... New Loaders are backwards compatible but not always forward compatible.  Does this need to be an optional param?

Reply:
Could potentially leave it off unless/until it's required?

I think with this change it's recommended that people who want to issue an external request should check the presence of the loader in the scope and use that, right?  Which should be external-capable, or even if it isn't then they can handle the error/404?

Reply:
Yeah I don't think it's necessary with the current state of things.  Tony mentioned it in his pr and I thought it would be nice to add, but also that checking an optional boolean just seems really weird.  I'll just remove it for now then.